### ISSUE TITLE
chat demo running error

### ISSUE THREAD

**Issue by surfingtomchen** [topfreegames/pitaya#126]
Environment: 
go 1.13.5
macos 10.15.2

I cloned the pitaya, follow the step to setup the dependancies by using `make setup` and cd the demo folder to run the following commands
```
docker-compose -f ../../testing/docker-compose.yml up -d etcd nats
go run main.go
```

everything is fine.

And I open the webpage  http://localhost:3251/web/

I can see the username , textfield to input and button to send. But after I input the text and push the send button nothing happened. 

I go back to the terminal of server, realize there is the error message:

>Failed to decode message: wrong packet type   source=pitaya

here is the log containing the error message

``` shell
INFO[0000] all modules started!                          source=pitaya
DEBU[0060] UserCount: Time=> 2019-12-29 16:05:48.418552 -0800 PST m=+60.006227350, Count=> 0, Error=> %!q(<nil>)  source=pitaya
DEBU[0120] UserCount: Time=> 2019-12-29 16:06:48.41908 -0800 PST m=+120.006548596, Count=> 0, Error=> %!q(<nil>)  source=pitaya
DEBU[0120] type: chat, servers: map[b35b640f-5a81-4169-94e5-d9fb6414be61:0xc0002ea000]  source=pitaya
DEBU[0179] New session established: Remote=127.0.0.1:62450, LastTime=1577664467  source=pitaya
DEBU[0179] Received data on connection with len 521      source=pitaya
ERRO[0179] Failed to decode message: wrong packet type   source=pitaya
DEBU[0179] Session closed, ID=1, UID=, IP=127.0.0.1:62450  source=pitaya
DEBU[0179] Session read goroutine exit, SessionID=1, UID=%!d(string=)  source=pitaya
DEBU[0179] Session handle goroutine exit, SessionID=1, UID=%!d(string=)  source=pitaya
DEBU[0180] UserCount: Time=> 2019-12-29 16:07:48.422466 -0800 PST m=+180.009728082, Count=> 0, Error=> %!q(<nil>)  source=pitaya
```

Did I miss anything? I've tried the nano sample which is working.

**Comment by felipejfc** [comment]
Hi @surfingtomchen 

This was my bad and I fixed it on 10f5ffb8b6217dec820b0d3365b3da186a00dfce, problem is that chat demo should use a ws acceptor, and I changed it to TCP recently. Should be fine now.

Best

**Comment by surfingtomchen** [comment]
it's ok now. Thanks.
